js/app.js

```diff
@@ -1,164 +1,183 @@
-    alert("But they are so close. I couldn't help but visit.");
-  } else {
-    //console.log ("Please answer with a yes or no");
-    alert("Please answer with a (y)es or (n)o");
-  }
-
-
-
-  let answer3 = prompt(
-    "Have I visited all the National Parks in Utah?"
-  )?.toLowerCase();
-
-  if (answer3 === "yes" || answer3 === "y") {
-    //console.log ("Yes! Capitol Reef has orchards that are known for their pies!");
-    alert("Yes! Capitol Reef has orchards that are known for their pies!");
-    playerScore++;
-  } else if (answer3 === "no" || answer3 === "n") {
-    //console.log ("But Capitol Reef features pies made in the National Park!");
-    alert("But Capitol Reef features pies made in the National Park!");
-  } else {
-    //console.log ("Please answer with a yes or no");
-    alert("Please answer with a (y)es or (n)o");
-  }
-
-
-
-  let answer4 = prompt(
-    "Have I visited all the National Parks in California?"
-  )?.toLowerCase();
-
-  if (answer4 === "yes" || answer4 === "y") {
-    //console.log ("Yes! Death Valley is one of the coolest places I have ever been.");
-    alert("Yes! Death Valley is one of the coolest places I have ever been.");
-    playerScore++;
-  } else if (answer4 === "no" || answer4 === "n") {
-    //console.log ("Oh no! California is a great road trip.");
-    alert("Oh no! California is a great road trip.");
-  } else {
-    //console.log ("Please answer with a yes or no");
-    alert("Please answer with a (y)es or (n)o");
-  }
-
-  let answer5 = prompt("Have I visited the Grand Canyon?")?.toLowerCase();
-
-  if (answer5 === "yes" || answer5 === "y") {
-    //console.log ("Yes, I watched a big snow storm roll into the canyon.");
-    alert("Yes, I watched a big snow storm roll into the canyon.");
-    playerScore++;
-  } else if (answer5 === "no" || answer5 === "n") {
-    //console.log ("I had to see the Grand Canyon!");
-    alert("I had to see the Grand Canyon!");
-  } else {
-    //console.log ("Please answer with a yes or no");
-    alert("Please answer with a (y)es or (n)o");
-  }
-
-
-//let outro = prompt("Thanks so much " + user + "! Welcome to my page!");
-//comment out outro due to new questions
-
-let correctAnswer = 27;
-let guesses = 4;
-
-while (guesses) {
-  let natGuess = prompt(
-    "Can you guess how many National Parks I have visited? Hint: there are only 63 National Parks."
-  );
-
-  guesses--;
-  if (natGuess > correctAnswer) {
-    alert("That number is too high.");
-  } else if (natGuess < correctAnswer) {
-    alert("That number is too low.");
-  } else if (natGuess == 27) {
-    alert("Correct! Congratulations!");
-    playerScore++;
-    break;
-  }
-  if (guesses === 0) {
-    alert("Only 4 guesses today! The number was " + correctAnswer);
+function fiveQuestions(answer) {
+  answer = answer.toLowerCase();
+  if (answer === "yes") {
+    return true;
+  } else if (answer === "no") {
+    return false;
```

Comment:
If you're only checking yes or no, you can make the correct answer an argument in the function instead of making everything a yes question.